packages/framework/src/evaluate/context.ts

```diff
@@ -40,5 +44,7 @@ export const createEvaluationContext = ({
   const app = {
     languages: applicationContext.application?.languages,
   };
-  return merge({}, { app, ensemble }, appInputs, screenInputs, context);
+  return merge({}, { app, ensemble }, appInputs, screenInputs, context, {
+    device: applicationContext.device,
```

Comment:
Can we pass this in instead of adding it here?

Reply:
where should need to pass ?

Reply:
By the context object - why did you add this here in the first place?